### ISSUE TITLE
Lack of Computational resource consumption in paper

### ISSUE THREAD

**Issue by greasebig** [zibojia/SENORITA#12]

**Comment by zibojia** [comment]
The final version has these information.

**Comment by greasebig** [comment]
Specifically, For the described two-stage training (CogVideoX-5B-I2V + ControlNet) with the given hyperparameters, what are the training duration (days/hours), VRAM usage, and GPU(s) used? Particularly interested in the details related to the Style Transfer task.

```text
We use CogVideoX-5B-I2V(Yang et al., 2024) as the base
model and integrate it with ControlNet to leverage the edited
first frame to guide the editing process. The batch size of
the editing model’s training is 32, the learning rate is 1e-5,
and weight decay is 1e-4. We train model for 2 epoch. We
sample 33 frames of the videos to train with a resolution
of 336×592 in first stage. Different from the first stage, we
use higher resolution of 448×768 and batch size of 16 in
stage two, fintuning with 1 epoch to help model edit high
resolution
